### ISSUE TITLE
Deploy as ROOT app in tomcat

### ISSUE THREAD

**Issue by jrichardsz** [openxava/xavaprojects#1]
Currently deploy applications in the same tomcat server is not the best way for a lot of reasons. 

## Context

I need to deploy my ox app as ROOT instead a custom name. 

## Process

1. Create a new ox project
2. Run createWar task
3. Rename the generated war to ROOT.war
4. Deploy in latest tomcat 9

## Expected result

If my tomcat is hosted in locahost:8085, this address must be return de ox app.

## Current result

locahost:8085 does not return anything and several errors are showed

## Possible Fix

Basically y change a few java class and add a new variable which works as **application name**

I did it but I need to improve it to perform a pull request.
- com.openxava.naviox.Modules
- com.openxava.naviox.web.NaviOXFilter
- com.openxava.naviox.web.NaviOXServlet
- /web/naviox/index.jsp

**Comment by javierpaniza** [comment]
Hi Richard,

Deploying OpenXava applications in the ROOT context is a nice improvement for OpenXava, but we should treat this issue in the Open Discussion forum (in English):
https://sourceforge.net/p/openxava/discussion/419689/

Or the Debate forum (in Spanish):
https://sourceforge.net/p/openxava/discussion/437015/

By now the official OpenXava forums are in SourceForge and we use GitHub just for the code.
